Packs/Tenable_io/Integrations/TenableioEventCollector/TenableioEventCollector.yml

```diff
@@ -0,0 +1,148 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Tenable.io Event Collector
+  version: -1
+configuration:
+- displaypassword: Access Key
+  additionalinfo: Tenable API access key.
+  name: access_key
+  required: true
+  hiddenusername: true
+  type: 9
+- displaypassword: Secret Key
+  additionalinfo: Tenable API secret key.
+  name: secret_key
+  required: true
+  hiddenusername: true
+  type: 9
+- display: Vulnerabilities Fetch Interval
+  additionalinfo: Fetch interval in minutes.
+  name: vuln_fetch_interval
+  required: true
+  type: 0
+- display: Severity
+  additionalinfo: The severity of the vulnerabilities to include in the export.
+  name: severity
+  options:
+  - info
+  - low
+  - medium
+  - high
+  - critical
+  required: false
+  type: 16
+- defaultvalue: 7 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days)
+  hidden: false
+  name: first_fetch
+  required: false
+  type: 0
+- additionalinfo: The maximum amount of audit logs to retrieve for each event type. For more information about event types see the help section.
+  defaultvalue: '1000'
+  display: Max Fetch
+  hidden: false
+  name: max_fetch
+  required: false
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+description: Tenable.io Event Collector integration.
```

Comment:
Let's rephrase this to something like "use Tenable.io to collect audit logs and vulns..."